src/LaravelDebugbar.php

```diff
@@ -905,7 +905,7 @@ protected function isJsonRequest(Request $request, Response $response)
 
         // Check if content looks like JSON without actually validating
         $content = $response->getContent();
-        if ($content !== false && in_array($content[0], ['{', '['], true)) {
+        if ($content && in_array($content[0], ['{', '['], true)) {
```

Comment:
```diff
-if ($content && in_array($content[0], ['{', '['], true)) {
+if (is_string($content) && in_array($content[0], ['{', '['], true)) {
```

Reply:
```diff
-if ($content && in_array($content[0], ['{', '['], true)) {
+if (is_string($content) && json_decode($content) !== null) {
```

I think we should use json_decode because may be json has contains space first or last

Reply:
https://github.com/barryvdh/laravel-debugbar/pull/1677